src/main/java/nextstep/subway/line/dto/SectionPath.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.subway.line.dto;
+
+import nextstep.subway.station.domain.Station;
+
+public class SectionPath {
+    private Station upStation;
+    private Station downStation;
+    private int distance;
+
+    private SectionPath(Station upStation, Station downStation, int distance) {
+        this.upStation = upStation;
+        this.downStation = downStation;
+        this.distance = distance;
+    }
+
+    public static SectionPath of(Station upStation, Station downStation, int distance) {
```

Comment:
```suggestion
    public static SectionPath of(Section section) {
```

[1. 함수 인자는 2개 이하로 작성하자](https://blog.sinsiway.com/31) 클린코드의 관점에서 인자를 줄여볼수 없을지 고민해보면 좋을것 같아요!

Reply:
좋은 아티클 공유해 주셔서 감사합니다 ! 
피드백 내용 확인하고 코드를 다시 돌아보니 
`SectionPath`의 `of` 메서드에서 받고자 하는 정보들은 `Section` 클래스에서 이미 다 갖고 있었네요 !
따라서 `SectionPath`에 `Section` 객체를 인자로 전달하면 `SectionPath` 클래스에서 필요한 데이터를 꺼내 사용하는 방식으로 개선하였습니다 
피드백 감사합니다 !! ☺️👍